js/glsl/scatter-vertex.glsl

```diff
@@ -50,7 +50,31 @@ attribute vec3 color_previous;
 void main(void) {
     vec3 origin = vec3(xlim.x, ylim.x, zlim.x);
     vec3 size_viewport = vec3(xlim.y, ylim.y, zlim.y) - origin;
+#ifdef USE_SPRITE
+    vec3 vector = v;
```

Comment:
vectors are only used for the direction of a 3d glyph/geo, to make quiver plots possible, we can ignore them